README.adoc

```diff
@@ -31,28 +31,29 @@ Plug 'raghur/vim-ghost', {'do': ':GhostInstall'}
 
 [source,vimscript]
 ----
-Plug 'roxma/nvim-yarp', {'cond': v:version == 800 && !has('nvim')}
-Plug 'roxma/vim-hug-neovim-rpc',  {'cond': v:version == 800 && !has('nvim')}
-
+" Only enabled for Vim 8 (not for Neovim).
+Plug 'roxma/nvim-yarp', v:version == 800 && !has('nvim') ? {} : { 'on': [], 'for': [] }
+Plug 'roxma/vim-hug-neovim-rpc', v:version == 800 && !has('nvim') ? {} : { 'on': [], 'for': [] }
```

Comment:
I tried using `cond:`, but it didn't seem to work for me. Found this in the wiki: https://github.com/junegunn/vim-plug/wiki/tips#conditional-activation

Reply:
I have this in my .vimrc

```
Plug 'roxma/nvim-yarp', {'cond': v:version == 800 && !has('nvim')}
Plug 'roxma/vim-hug-neovim-rpc',  {'cond': v:version == 800 && !has('nvim')}
```

Reply:
Yeah, it runs, but I  don't think it actually skips on nvim (or old vim).

For example, with this vimrc in `~/tmp/vimrc`:

```viml
" Add vim-plug dir to vim runtimepath (already there for nvim).
exe 'set rtp+=' . $XDG_DATA_HOME . '/nvim/site'

call plug#begin('~/tmp/plugged')    " Load plugins with vim-plug.

Plug 'roxma/nvim-yarp', {'cond': v:version == 800 && !has('nvim')}
Plug 'roxma/vim-hug-neovim-rpc',  {'cond': v:version == 800 && !has('nvim')}

call plug#end()                                   " Initialize plugin system
```

Running:

```bash
nvim -u ~/tmp/vimrc +PlugUpdate +PlugStatus
```

Gives this:

<img width="248" alt="image" src="https://user-images.githubusercontent.com/15943089/50555605-b6b6c500-0cc6-11e9-9dc9-03b071d79cbb.png">

---

If you change the above example to:

```diff
-Plug 'roxma/nvim-yarp', {'cond': v:version == 800 && !has('nvim')}
+Plug 'roxma/nvim-yarp', v:version == 800 && !has('nvim') ? {} : { 'on': [], 'for': [] }
-Plug 'roxma/vim-hug-neovim-rpc',  {'cond': v:version == 800 && !has('nvim')}
+Plug 'roxma/vim-hug-neovim-rpc', v:version == 800 && !has('nvim') ? {} : { 'on': [], 'for': [] }
```

you get:

<img width="373" alt="image" src="https://user-images.githubusercontent.com/15943089/50555618-de0d9200-0cc6-11e9-907c-610c4abf515c.png">

---

I actually only worked this out because I saw this snippet and thought it looked awesome, so I wanted to steal it...

Reply:
Thank you for taking the time to put that together.

You're absolutely right - I should have checked harder. So the 'cond' key is something that I hacked together in my .vimrc for deferred plugin loading. It's been so long that I totally forgot about it..

```
function! DeferPluginLoad(name, ...)
    " echo a:000
    if !has("vim_starting")
        return
    endif
    let opts = get(a:000, 0, {})
    let cond = 1
    if has_key(opts, 'cond')
        let cond = opts['cond']
    endif
    let opts = extend(opts, { 'on': [], 'for': [] })
    Plug a:name, opts
    if cond
        let g:deferredPlugins = g:deferredPlugins + split(a:name, '/')[1:]
    endif
endfunction

command! -nargs=* DeferPlug call DeferPluginLoad(<args>)
```

Reply:
Cool! `cond` seems like something that would be worth upstreaming to vim-plug FWIW.